lib/aca_entities/serializers/xml/medicaid/atp.rb

```diff
@@ -72,18 +72,14 @@
 require_relative "atp/insurance_application_assister_reference"
 require_relative "atp/physical_household"
 require_relative "atp/insurance_application_assister_association"
-require_relative "atp/referral_activity_eligibility_reason_reference"
-require_relative "atp/referral_activity_receiver_reference"
-require_relative "atp/referral_activity_sender_reference"
-require_relative "atp/referral_activity_status"
 require_relative "atp/referral_activity"
+require_relative "atp/additional_referral_activity"
 require_relative "atp/document_expiration_date"
 require_relative "atp/document_number"
 require_relative "atp/document_person_identification"
 require_relative "atp/immigration_document"
 require_relative "atp/eligibility_basis_determination"
 require_relative "atp/lawful_presence_status_eligibility"
-require_relative "atp/insurance_applicant_lawful_presence_status"
```

Comment:
Medicaid gateway will have it's own PR - and we'll kinda need to get it done before we start work in Enroll.

The reason I moved the file is so that we move more toward the 'library' approach of selective inclusion.

To that end, I've tried to move the requires of the 'child' xml elements into the 'parent' XML element files.

As a concrete case,
`atp/insurance_applicant_lawful_presence_status` is the child element, and it's really only used by the `InsuranceApplicant` element in `atp/insurance_applicant.rb`.

I've tried to, where appropriate, make similar movement of files and requires.